3.3/test/run

```diff
@@ -84,7 +89,8 @@ wait_for_cid() {
   local sleep_time=1
   local attempt=1
   local result=1
-  info "Waiting for application container to start"
+  local cargs=${CONTAINER_ARGS:-}
+  info "Waiting for application container to start $cargs"
```

Comment:
@rhcarvalho - actually now that I look at it, no purpose. I needed it here: https://github.com/openshift/sti-python/pull/13/files#diff-4d94a409b126ccb3db3b109be441a31cR57 and I ended up using the same thing everywhere else. You're welcome to submit a fix to remove it :-)